### PR TITLE
RTD V2 Config

### PR DESCRIPTION
**Pull request by nabobalis**
Moved to RTD v2 and updated 32bit build. 

RTD built on my fork: https://readthedocs.org/projects/mysunpy/builds/

### CODE REVIEW

**Comment by nabobalis** [comment]
The astropy_helpers changes are more changes to master that I pulled in.

**Comment by nabobalis** [comment]
Not sure why Circle did not run.

**Comment by nabobalis** [comment]
Seems like some online server issues.

File: .circleci/config.yml [hunk 1]
```diff
@@ -68,25 +68,28 @@ jobs:
...
       - checkout
       - run: *skip-check
       - run:
-          name: yum dependencies
-          command: yum install -y freetype freetype-devel libpng-devel tk
+          name: apt dependencies
+          command: |
+            export DEBIAN_FRONTEND="noninteractive"
+            apt-get update
+            apt-get install -y libfreetype6 libfreetype6-dev libpng-dev tk pkg-config python3-pip git openssh-client
```
Comment (Cadair) [inline .circleci/config.yml]: Is this before the clone stage?

File: .circleci/config.yml [hunk 2]
```diff
@@ -68,25 +68,28 @@ jobs:
...
       - run:
           name: Install dependencies one
-          command: /opt/python/cp36-cp36m/bin/pip install --progress-bar off numpy==1.15.3 scipy astropy!=3.1.1 matplotlib==1.5.3
+          command: pip3 install cython numpy scipy astropy matplotlib | cat
```
Comment (Cadair) [inline .circleci/config.yml]: Why the pipe to cat ooi?

File: .readthedocs.yml [hunk 3]
```diff
@@ -1,9 +1,11 @@
...
-  pip_install: false
-  setup_py_install: true
+   version: 3.7
+   install:
+      # Use this to override dep versions.
+      - requirements: docs/rtd_requirements.txt
+      - method: pip
+        path: .
+        extra_requirements:
+           - all
```
Comment (Cadair) [inline .readthedocs.yml]: docs?

File: docs/rtd_requirements.txt [hunk 4]
```diff
@@ -1,23 +1,3 @@
...
-asdf
-beautifulsoup4
-requests
-python-dateutil
-ruamel.yaml
-sphinx
-sunpy-sphinx-theme
-sphinx-gallery
-sphinx-astropy
 git+https://github.com/hawkowl/towncrier
```
Comment (Cadair) [inline docs/rtd_requirements.txt]: we can go back to release towncrier now as a new version was pushed a couple of weeks ago

**Pull request merged**